oldlog.go

```diff
@@ -9,9 +9,10 @@ import (
 	tracer "github.com/ipfs/go-log/tracer"
```

Comment:
In a subsequent changeset, we should really change the name of this file. Not here, because it would muddy the diff.